payments-core/src/main/java/com/stripe/android/paymentsheet/model/StripeIntentValidator.kt

```diff
@@ -24,10 +24,13 @@ internal class StripeIntentValidator {
                 )
             }
             stripeIntent is PaymentIntent &&
-                stripeIntent.status != StripeIntent.Status.RequiresPaymentMethod -> {
+                (
+                    (stripeIntent.status != StripeIntent.Status.RequiresPaymentMethod) &&
+                        (stripeIntent.status != StripeIntent.Status.RequiresAction)
+                    ) -> {
```

Comment:
Do we need to make this change for Setup Intents too, below?

Reply:
yes, done